### PR TITLE
docs(examples): added pupperender

### PR DESCRIPTION
**Pull request by LasaleFamine**
Related #1289.

Let me know if you want to change the description. @ebidel

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ NODE_PATH=../ node examples/search.js
 ## Web scraping
 - [Puppetron](https://github.com/cheeaun/puppetron) - Demo site that shows how to use Puppeteer and Headless Chrome to render pages. Inspired by [GoogleChrome/rendertron](https://github.com/GoogleChrome/rendertron).
  - [Thal](https://medium.com/@e_mad_ehsan/getting-started-with-puppeteer-and-chrome-headless-for-web-scrapping-6bf5979dee3e "An article on medium") - Getting started with Puppeteer and Chrome Headless for Web Scraping.
+ - [pupperender](https://github.com/LasaleFamine/pupperender) - ExpressJs middleware that checks the User-Agent header of incoming requests, and if it matches one of a configurable set of bots, render the page using Puppeteer. Useful for PWA rendering.
```
Comment (ebidel) [inline examples/README.md]: ExpressJs -> Express

Comment (ebidel) [reply examples/README.md]: bots -> search crawlers

maybe?

**Review by ebidel — CHANGES_REQUESTED** [review]

File: examples/README.md (hunk 1 above)
Comment (LasaleFamine) [reply examples/README.md]: Yeah, I was thinking to add another main area. I'll go for search crawlers, we can change it later.

Comment (ebidel) [reply examples/README.md]: Yea but 'Puppetron' is also for rendering. Let's not add a new category and rename the existing title `## Rendering and web scraping` ?

Comment (LasaleFamine) [reply examples/README.md]: Make sense 👍

Comment (LasaleFamine) [reply examples/README.md]: PS: I fixed a little space left at the start of the Thal's article line.

**Review by ebidel — APPROVED** [review]

**Pull request merged**